ckanext/canada/i18n/ckanext-canada.pot

```diff
@@ -2233,3 +2233,5 @@ msgstr ""
 msgid "Remember me"
 msgstr ""
 
+msgid "Date format incorrect. Expecting YYYY-MM-DD"
+msgstr "Date format incorrect. Expecting YYYY-MM-DD"
```

Comment:
pot files don't include translations. they're translation templates for new languages and for updating existing translations only. Also, they should only be updated by using the `setup.py extract_messages` command